backend/frontend/templates/frontend/auth/page_login.html

```diff
@@ -0,0 +1,30 @@
+{% extends "frontend/auth/base.html" %} {% block content %}
+<div class="mx-auto w-1/3 p-6 bg-white rounded">
+  <h1 class="{{ styles.heading1 }}">Login</h1>
+  <hr class="my-2" />
+  {% if messages %}
+    {% for message in messages %}
+        <p>
```

Comment:
a paragraph inside a paragraph? This is a new use of paragraph I have not come across before... I'm sure it'll render fine but, what are you actually trying to do here? 

Should this be an unordered list? Or a paragraph in a div? Or what?